app/javascript/components/Reservations.js

```diff
@@ -4,11 +4,13 @@ import { fetchReservationsApi } from '../redux/features/reservations/reservation
 import { fetchSections } from '../redux/features/sections/sectionsSlice';
 import '../../assets/stylesheets/reservations.css';
 import { CiUser } from "react-icons/ci";
+import NavBar from './NavBar';
 
 function Reservations() {
```

Comment:
- [ ] According to the project requirements for teams of 2:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/df5a2d65-8e42-4ea5-94b5-222ddbb99d19)

I can't see any login page to type my name to create a session.  This means all users share the same session. 

Kindly, make it so each user has to type their username to log into the website (tip: In order to persist the user, you can use localStorage).  Make sure there is a way to logout so more users can use the app (in order to logout, you delete the localstorage and reload the page). 👍 

- [ ] All your users share the same reservations page:

I created several reservations for both "Betty" and "Juan" and they appear on the same page:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/c83fc8ed-5f88-454b-ab8e-bb77f27549d1)

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/40f4c978-b66f-47a0-a45e-28b35418e120)

Kindly, make it so each user has their own reservations page.  The page should display only the reservations for that user. 👍 

_tip:  The reservations feature is not a requirement for a team of 2 people.  Therefore, you don't have to include it. However, if you implement it, it needs to work as it was intended._

_tip 2: The work is already there because there is already an API endpoint to create users.  You would just need to implement the frontend for that feature._